priority_variables_transform/JHS-ingest/hist_my_inf.yaml

```diff
@@ -4,6 +4,8 @@
       slot_derivations:
         associated_participant:
           populated_from: phv00127670
+        age_at_condition_start:
+          expr: case(phv00127708 == 'Yes', {phv00127710} * 365)
```

Comment:
All of the rest of these JHS `age_at_condition_start:` statements with `expr: case(...` are broken the same way. The need to be:
```
expr: case(({phv00127708} == 'Yes', {phv00127710} * 365))
```
They need the double parens and the phv#s need to be bracketed. I'm not going to call each of them out here but ping me if you need help.